lib/prawn/text/formatted/arranger.rb

```diff
@@ -212,6 +219,8 @@ def font_style(styles)
 
         private
 
+        attr_accessor :finalized
```

Comment:
Matz agreed that this warning is wrong and it's going to be removed in a future version.

https://bugs.ruby-lang.org/issues/10967